### PR TITLE
SC-10. Updated upcoming birthdays functionality

### PR DESCRIPTION
**Pull request by yuratouch**
Updated get_upcoming_birthdays() method. 
Add additional argument to implement as a variable for the given days range.

### CODE REVIEW

File: src/modules/contact_book.py [hunk 1]
```diff
@@ -21,43 +21,32 @@ def delete(self, name):
                 self.data.pop(contact_name)
                 break
 
-    # TODO: Add extra parameter days_to for method get_upcoming_birthdays
-    # def get_upcoming_birthdays(self, days_to: int = 7):
-    def get_upcoming_birthdays(self):
+    def get_upcoming_birthdays(self, next_days: int = 7):
```
Comment (anedost) [inline src/modules/contact_book.py]: set return data type  (->)

File: src/handlers/contacts_handler.py [hunk 2]
```diff
@@ -88,5 +88,20 @@ def show_birthday(args: list, book) -> str:
     return "Contact not found."
 
 
-def show_upcoming_birthdays(book) -> list:
-    return book.get_upcoming_birthdays()
+def show_upcoming_birthdays(args:list, contact_book:ContactBook):
```
Comment (anedost) [inline src/handlers/contacts_handler.py]: set return data type (->)

File: main.py [hunk 3]
```diff
@@ -47,7 +47,7 @@ def main():
             print(show_birthday(args, book))
 
         elif command == "birthdays":
-            print(show_upcoming_birthdays(book))
+            print(show_upcoming_birthdays(args, book))
```
Comment (anedost) [inline main.py]: if you merge SC-15 first, this line have a conflict, FYI

File: src/modules/contact_book.py (hunk 1 above)
Comment (yuratouch) [reply src/modules/contact_book.py]: data type added

**Comment by yuratouch** [comment]
All mentioned issues are fixed. 
PR is ready for review. 
@AlexNedos

**Review by anedost — APPROVED** [review]

**Pull request merged**